webServer.js

```diff
@@ -270,7 +270,7 @@ function start() {
 
       if (assets.gennedFileNames === undefined) {
         var fileArray = [];
-        assets.files.map(function (fileName, index) {
+        assets.files.map(function (fileName) {
```

Comment:
Could this change be split into a separate commit?

One commit for the eslint file & changes to reflect the eslint enforcement, and a secondary commit for other tidying?
Or is this simply part of the eslint enforcement?

Reply:
I think its all part of eslint personally and we'd be splitting hairs to separate it out.

Reply:
Ah cool, that's fine then :)